teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(id, age, specialization, name, parent_permission: true)
+    super(id, age, name, parent_permission: parent_permission)
+    @specialization = specialization
+  end

```

Comment:
- [ ] Kindly remove the `id` parameter from the initialize method and from the super call.

- [ ] Name parameter should have a default value of `Unknown` in the initialize method.